src/pytsql/tsql.py

```diff
@@ -154,8 +154,6 @@ def _split(code: str, isolate_top_level_statements: bool = True) -> List[str]:
             " and is preventing the use of the faster C++ parser."
         )
 
-    logging.info("Started SQL script parsing")
```

Comment:
Some more general suggestions:
1. We could also log those as debug statements
2. We could use a dedicated named logger for `pytsql` like `logger = logging.getLogger("pytsql")`, and then people also have the opportunity to disable all `pytsql` logs

That said, I don't find those statements that helpful, so I am also fine with removing them.

Reply:
As we discussed, I'd turn them into `debug` even when they're not too helpful and give the logger a name. Improving the content of the logging itself can be done in a separate effort.

Reply:
Good question. Not sure. My guess is that you don't need it in the `__init__.py`.